dockernet/tests/integration_tests.bats

```diff
@@ -199,6 +193,56 @@ setup_file() {
   assert_equal "$stake_diff" "1"
 }
 
+@test "[INTEGRATION-BASIC-$CHAIN_NAME] LSM liquid stake" {
+  if [[ "$CHAIN_NAME" != "LSM" ]]; then
```

Comment:
After the LSM is an accepted module to the hub, then GAIA in general will have LSM as part of it... will we still have the LSM as a separate docker build at that point?  Will this test update in the future or do we plan to have a vanilla GAIA and an LSM specific hub as separate dockers even after LSM is accepted?

Reply:
imo once lsm is part of the gaia codebase, we should just use the latest gaia binary, rather than have two dockers

Reply:
maybe we can comment around areas of dockernet that will need to change once lsm is in the standard gaia codebase

Reply:
perhaps we can check whether the chain has LSM enabled in a different way, maybe by checking whether it has a x/staking globalCap param and that it's nonzero

Reply:
once LSM is in gaia, we'll remove the LSM host zone entirely and update gaia to the latest version. I don't think we need to flag it now - it'll just be a find + delete of everything LSM in dockernet